tests/test_norm_ops.py

```diff
@@ -74,6 +74,7 @@ def test_accuracy_groupnorm(N, C, H, W, num_groups, dtype):
     gems_assert_close(res_bias_grad, ref_bias_grad, dtype, reduce_dim=N * HW)
 
 
+@pytest.mark.skip("triton_musa unsupport")
```

Comment:
please figure out the reason why not support LayerNorm, cause group norm with similar algorithm is supported.

Reply:
it may be a compiler bug and under locating, we will support this op later after the bugs fixed.